ajax/libs/logdown.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "logdown.js",
+  "filename": "logdown.min.js",
+  "version": "3.2.2",
+  "description": "Debug utility with markdown support that runs on browser and server",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/caiogondim/logdown"
+  },
+  "keywords": [
+    "log",
+    "markdown",
+    "debug",
+    "console",
+    "info",
+    "error",
+    "logging"
+  ],
+  "author": "Caio Gondim <me@caiogondim.com> (https://caiogondim.com)",
+  "homepage": "http://caiogondim.github.io/logdown.js/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brightcove/videojs-thumbnails.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/logdown*.(js|map)"
```

Comment:
We use exclude mode when the basePath is `dist`.
Let's use exclude mode.
Thanks.

Reply:
I little stuck with this pattern. Could you help to to figure put what it should be. Thanks